model_zoo/pytorch/nanogpt/prepare.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2023 The DLRover Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+import pickle
+
+import numpy as np
+
+
+def prepare_dataset(input_file_path):
+    with open("dlrover/model_zoo/pytorch/nanogpt/data/input.txt", "r") as f:
```

Comment:
Where is `input.txt`?

Reply:
Thank you for pointing out this mistake. This was an oversight on my part. I had initially hard-coded the file path for testing purposes and forgot to change it back. 

The correct code should utilize the function's parameter `input_file_path` as shown below:

```python
def prepare_dataset(input_file_path):
    with open(input_file_path, "r") as f: